pathspider/plugins/evilbit.py

```diff
@@ -0,0 +1,111 @@
+from scapy.all import IP  # pylint: disable=E061
+from scapy.all import IPv6  # pylint: disable=E061
+from scapy.all import UDP  # pylint: disable=E061
+from scapy.all import TCP  # pylint: disable=E061
+from scapy.all import DNS  # pylint: disable=E061
+from scapy.all import DNSQR  # pylint: disable=E061
+from scapy.all import RandShort  # pylint: disable=E061
+
+import pathspider
+from pathspider.base import PluggableSpider
+from pathspider.forge import ForgeSpider
+from pathspider.chains.dns import DNSChain
+from pathspider.chains.basic import BasicChain
+from pathspider.chains.tcp import TCPChain
+from pathspider.chains.evil import EvilChain
+from pathspider.chains.base import Chain
+from pathspider.chains.tcp import TCP_SYN, TCP_SA
+
+
+class EvilBit(ForgeSpider, PluggableSpider):
+
+    name = "evilbit"
+    description = "Evil bit connectivity testing (RFC 3514)"
+    #version = pathspider.__version__
```

Comment:
Uncomment this, I don't remember where the version is now but setup.py will tell you.